great_expectations/cli/datasource.py

```diff
@@ -1220,117 +1220,79 @@ def _get_batch_kwargs_from_generator_or_from_file_path(
 def _get_batch_kwargs_for_sqlalchemy_datasource(
     context, datasource_name, additional_batch_kwargs=None
 ):
-    msg_prompt_query = """
-Enter an SQL query
-"""
+    msg_prompt_single_or_multiple_data_assets = """Does the expectation suite involve data from a single table or multiple tables?
+    1. Single table
+    2. Custom query involving multiple tables
+    """
 
-    msg_prompt_enter_data_asset_name = (
-        "\nWhich table would you like to use? (Choose one)\n"
-    )
+    data_asset_name = None
+    sql_query = None
 
-    msg_prompt_enter_data_asset_name_suffix = (
-        "    Do not see the table in the list above? Just type the SQL query\n"
+    single_or_multiple_data_asset_selection = click.prompt(
+        msg_prompt_single_or_multiple_data_assets,
+        type=click.Choice(["1", "2"]),
+        show_choices=False,
     )
 
+    if single_or_multiple_data_asset_selection == "1":  # single table
+        schema_name = click.prompt("Please provide the schema name of the table")
```

Comment:
It would be great to have default schema names per database engine. Otherwise, this might be difficult for users. For example, the default schema for sqlite is "main", which none of us knew until we had to look it up while reviewing this PR.

Reply:
@eugmandel if a user wants to create an expectation suite on a table, they'd know which schema it belongs to. Can you elaborate on how a default schema will help the end user here? 🤔

Reply:
added via 13576f7